pallets/cash/src/internal/events.rs

```diff
@@ -108,6 +108,13 @@ pub fn receive_event<T: Config>(
     event: ChainLogEvent,
     signature: ValidatorSig,
 ) -> Result<(), Reason> {
+    log!(
```

Comment:
Probably too chatty (?) not sure if we want to address this in some broader way though where during sync you don't show all the logs and you start showing them when you catch up.

Reply:
Yeah, I think we'll remove many of them once we get things stabilized.